### DISCUSSION TITLE
How do pull/merge requests and branches work?

### DISCUSSION THREAD

**Discussion by stardriftfx** [UWAppDev/community#5] [Q&A] [answered]
Working with a team and we're using Github for collaboration.

**Comment by XinyiXiang** [comment, marked as answer]
I would recommend start by looking at [GitHub Docs](https://docs.github.com/en/github/collaborating-with-pull-requests/incorporating-changes-from-a-pull-request/merging-a-pull-request) for pulling and merging requests. 

As an additional note, if you'd like to learn more about branches or git in general, go ahead and check out [GitStarted](https://www.youtube.com/watch?v=WM8msxE7kZY), our club produced tutorial video on git as well.
